userspace/libscap/engine/nodriver/nodriver_public.h

```diff
@@ -1,24 +1,17 @@
 /*
 Copyright (C) 2022 The Falco Authors.
-
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-
     http://www.apache.org/licenses/LICENSE-2.0
-
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-#include "bpf_public.h"
-#include <stdlib.h>
-
-const char *scap_get_bpf_probe_from_env()
-{
-	return getenv(SCAP_BPF_PROBE_ENV_VAR_NAME);
-}
+#pragma once
 
+#define NODRIVER_ENGINE "nodriver"
+#define NODRIVER_ENGINE_LEN 9
```

Comment:
Also, specifying the length to just do a `strncmp(x, X_ENGINE, X_ENGINE_LEN)` is exactly equal to just call `strcmp(x, X_ENGINE)`; i think we can save us from defining the len everywhere and just use plain old `strcmp`.